helm/hpcc/templates/_helpers.tpl

```diff
@@ -599,7 +599,9 @@ Check dll mount point, using hpcc.changeMountPerms
 */}}
 {{- define "hpcc.checkDllMount" -}}
 {{- if .root.Values.storage.dllStorage.forcePermissions | default false }}
-{{ include "hpcc.changeMountPerms" (dict "root" .root "volumeName" "dllstorage" "volumePath" "/var/lib/HPCCSystems/queries") }}
+{{- $volumeName := (.root.Values.storage.dllStorage.plane | default "dll") -}}
+{{- $volumePath :=  include "hpcc.getVolumeMountPrefix" (dict "root" .root "me" .root.Values.storage.dllStorage "name" "dll" "path" "queries") }}
+{{ include "hpcc.changeMountPerms" (dict "root" .root "volumeName" $volumeName "volumePath" $volumePath) }}
```

Comment:
@xwang2713 - re. seperate JIRA - I mean changes like this - which aren't specific to NFS, and affects other systems too (anything that needs forcePermissions)